debian/changelog

```diff
@@ -1,3 +1,9 @@
+mongo-c-driver (1.24.2-2) UNRELEASED; urgency=medium
+
+  * use system uft8proc, ensure bundled source is not in .orig.tar.gz
```

Comment:
```suggestion
  * use system utf8proc, ensure bundled source is not in .orig.tar.gz
```

Including utf8proc is on the master branch and is planned in 1.25.0. Including utf8proc has not been released in 1.24.2. Should this PR be updated to remove references to utf8proc?

Reply:
Disregard, after discussion I now understand this is not suggesting utf8proc changes are includes in 1.24.2.